scripts/updateTranslationPath.ts

```diff
@@ -0,0 +1,46 @@
+import child from 'child_process'
+import fs from 'fs/promises'
+import util from 'util'
+
+const exec = util.promisify(child.exec)
+const translationsDir = 'src/assets/translations'
+
+export const updateTranslationPath = async (): Promise<void> => {
+  const previousPath = process.argv[2]
+  const newPath = process.argv[3]
+
+  const splitPreviousPath = previousPath.split('.')
+  const splitNewPath = newPath.split('.')
+
+  const oldKey = splitPreviousPath.pop()
+  const newKey = splitNewPath.pop()
```

Comment:
Mutative to the original array and returns the popped item, which is exactly what we want